site/content/en/docs/user/stages-configuration.md

```diff
@@ -61,6 +62,15 @@ and the changes that will be made to the resource when the stage is applied.
 The `next` field allows users to define the new state of the resource using the `statusTemplate` field,
 modify the `finalizers` of the resource, and even `delete` the resource.
 
+It is worth noting that there is no dedicated field for arranging the execution order if multiple stages of a resource type are provided.
+The execution order of stages can be naturally controlled by utilizing `selector.matchExpressions` and `next` field.
```

Comment:
```suggestion
The execution order of stages can be controlled by utilizing `selector.matchExpressions` and `next` field.
```
`naturally` may cause an intention to ask what is natural. Could we remove it?

Reply:
"naturally" is not necessary here. Thanks for improving the writing!